d2bs/kolbot/libs/bots/Summoner.js

```diff
@@ -4,7 +4,20 @@
 *	@desc		kill the Summoner
 */
 
-function Summoner() {
+function Summoner () {
+	// return the script position in the list of scripts to be ran
+	this.scriptIndex = function (script) {
```

Comment:
add a feature in Loader.js where the for loop for loading scripts occurs, you can save the index as a global in the loader object and add some function to get next script or get script by index + 1